test/algconvergence/ode_rosenbrock_tests.jl

```diff
@@ -273,6 +273,23 @@ import LinearSolve
     sol = solve(prob, ROS34PW3())
     @test length(sol) < 20
 
+    ### ROS34PRw
+    prob = prob_ode_linear
+
+    sim = test_convergence(dts, prob, ROS34PRw())
+    @test sim.𝒪est[:final]≈4 atol=testTol
```

Comment:
Shouldn't it be third-order accurate?
```suggestion
    @test sim.𝒪est[:final]≈3 atol=testTol
```